app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::CommentsController < ApplicationController
+  before_action :authenticate_user!, only: [:create]
+
+  def create
+    comment = current_user.comments.create(comment_params)
+    if comment.persisted?
+      render json: { comment: comment }, status: 201
+    else
+      render json: { message: comment.errors.full_messages.to_sentence }, status: 422
+    end
+  end
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(:content, :post_id)
+  end
```

Comment:
prob need to include username or email too in the comment params

Reply:
Good thinking! Fixed it :)